### PR TITLE
fix: when create user add additional keys

### PR DESCRIPTION
**Pull request by liubaterehova**

### CODE REVIEW

**Review by apsevkin — APPROVED** [review]

File: src/auth/auth.service.ts [hunk 1]
```diff
@@ -61,6 +61,10 @@ export class AuthService {
           status: 'incomplete',
           identityAccessKey: null,
         },
+        birthDate: new Date(),
```
Comment (apsevkin) [inline src/auth/auth.service.ts:64]: why do we set birthDay as a current date?

Comment (apsevkin) [reply src/auth/auth.service.ts:64]: Maybe it would be better to set it as an empty string or null?

**Review by apsevkin — CHANGES_REQUESTED** [review]

File: src/auth/dto/account.dto.ts [hunk 2]
```diff
@@ -21,6 +21,14 @@ export class AccountDto {
 
   public verificationStatus: RehiveKYCStatus;
 
+  public birthDate: Date;
```
Comment (apsevkin) [inline src/auth/dto/account.dto.ts:24]: I guess here should be union type Date | null

**Review by apsevkin — CHANGES_REQUESTED** [review]

File: src/auth/dto/account.dto.ts (hunk 2 above)
Comment (liubaterehova) [reply src/auth/dto/account.dto.ts:24]: It's not necessary to add null, cause any object can be null

**Review by apsevkin — APPROVED** [review]

**Pull request merged**